### PR TITLE
chore(docs): update doc

### PR DESCRIPTION
**Pull request by mgermerie**
## Description

Update the docs content

### CODE REVIEW

**Comment by mgermerie** [comment]
This PR follows the work done in PR #1526 and shall be rebased after PR #1526 is merged.

File: src/Controls/GlobeControls.js [hunk 1]
```diff
@@ -147,8 +147,9 @@ let previous;
  *
  * @class      GlobeControls
  * @param      {GlobeView}  view the view where the control will be used
- * @param      {CameraTransformOptions}  targetCoordinate the target looked by camera, at initialization
- * @param      {number}  range distance between the target looked and camera, at initialization
+ * @param      {object} placement
```
Comment (gchoqueux) [inline src/Controls/GlobeControls.js]: it's CameraTransformOptions

File: src/Controls/GlobeControls.js [hunk 2]
```diff
@@ -1161,7 +1162,7 @@ class GlobeControls extends THREE.EventDispatcher {
      * @param      {number}   [params.zoom]   zoom
      * @param      {number}   [params.scale]   scale
      * @param      {boolean}  isAnimated  Indicates if animated
-     * @return     {Promise}  A promise that resolves when transformation is oppered
+     * @return     {Promise}  A promise that resolves when transformation is opened
```
Comment (gchoqueux) [inline src/Controls/GlobeControls.js]: `the transformation is complete`

File: src/Renderer/Camera.js [hunk 3]
```diff
@@ -59,16 +62,34 @@ function updatePreSse(camera, height, fov) {
     }
 }
 
+/**
+ * Wrapper around Three.js camera to expose some geographic helpers.
+ *
+ * @param   {string}                crs                                     The CRS of Three.js coordinates.
```
Comment (gchoqueux) [inline src/Renderer/Camera.js]: crs is `the camera's coordinate projection system`

File: src/Renderer/Camera.js [hunk 4]
```diff
@@ -59,16 +62,34 @@ function updatePreSse(camera, height, fov) {
...
+ * @param   {number}                width                                   The width (in pixels) of the view the camera is
+ * associated to.
+ * @param   {number}                height                                  The height (in pixels) of the view the camera is
+ * associated to.
+ * @param   {object}                [options]                               Options for the camera.
+ * @param   {object}                [options.cameraThree]                   A custom Three.js camera object to wrap around.
```
Comment (gchoqueux) [inline src/Renderer/Camera.js]: `options.cameraThree` is `{THREE.Camera}`

File: src/Renderer/Camera.js [hunk 5]
```diff
@@ -59,16 +62,34 @@ function updatePreSse(camera, height, fov) {
...
+ * `CAMERA_TYPE.ORTHOGRAPHIC` must cover initially.
+ * @constructor
+ */
 function Camera(crs, width, height, options = {}) {
     Object.defineProperty(this, 'crs', { get: () => crs });
 
-    if (options.camera) {
-        this.camera3D = options.camera;
+    if (options.isCamera) {
+        console.warn('options.camera parameter is deprecated. See the documentation of Camera.');
```
Comment (gchoqueux) [inline src/Renderer/Camera.js]: add in message :  by what needs to be done to make it work

**Review by gchoqueux — COMMENTED** [review]
Good

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**